gulpfile.babel.js

```diff
@@ -60,7 +61,31 @@ gulp.task('webpack', (cb) => {
   });
 });
 
-gulp.task('serve', () => {
+    
+ 
+// Setting up the test task 
+gulp.task('protractor', ['serve'], function(callback) {
+    gulp
+        .src(['example_spec.js'])
+        .pipe(gulpProtractorAngular({
+            'configFile': 'protractor.conf.js',
+            'debug': false,
+            'autoStartStopServer': true
+        }))
+        .on('error', function(e) {
+            console.log(e);
+        })
+        .on('end', callback);
+});
+
+
+gulp.task('e2e', ['protractor'], function() {
+  process.exit();
```

Comment:
In the previous commit I have serve.exit() [from the browser-sync API](https://www.browsersync.io/docs/api/) instead, and it doesn't stop the server. Exiting the process seemed to be the only option.